tools/gen_bazel

```diff
@@ -56,6 +56,7 @@ gn_args_base = {
     'enable_perfetto_traced_perf': 'false',
     'perfetto_force_dcheck': '"off"',
     'enable_perfetto_llvm_demangle': 'true',
+    'enable_perfetto_winscope': 'false',
     # TODO(lalitm): we are disabling these temporarily because Studio cannot
     # build them and the we don't have a way to enable/disable these properly
     # at the Bazel level.
```

Comment:
Since you're here, please remove the etm ones as well and do the same as above (add !perfetto_build_generator in the gn args above).